ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_tensor.py

```diff
@@ -1,8 +1,4 @@
-# global
-import numpy as np
```

Comment:
Can you bring these imports back as they are not related to your changes?

Reply:
done